android/app/src/main/java/org/ocua/parity/tasks/LoadCurrentLeague.java

```diff
@@ -41,7 +41,6 @@ protected Long doInBackground(String... strings) {
             resString = EntityUtils.toString(response.getEntity());
             json = new JSONObject(resString);
         } catch (Exception e) {
-            this.dialog.setMessage(resString);
```

Comment:
I guess this isn't useful eh and actually masks the exception?

Reply:
Yeah, but the app was crashing because before it could print the stack trace from the original exception. I didn't dive into it, but it was something about that function being called from the wrong thread.